src/mono/netcore/System.Private.CoreLib/README.md

```diff
@@ -0,0 +1,8 @@
+# System.Private.CoreLib for Mono Runtime
+This folder contains Mono runtime specific build of System.Private.CoreLib.dll. The actual implementation is located in two folders. The code which has hard dependency on Mono VM implementation can be found under [src](src/) and common code in [shared](/src/libraries/System.Private.CoreLib/src/README.md).
```

Comment:
Would it be better if this file just said:
```
This folder contains Mono runtime specific portion of System.Private.CoreLib.dll. See `src\libraries\System.Private.CoreLib\src\README.md` for more details.
```
And have everything else in the readme under libraries? It will avoid duplication.

Reply:
Yep, changed it to be this way